### PR TITLE
LSR-1065 Support Geolocation Field

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+import java.text.ParseException;
+
+/**
+ * @author Víctor Galán Grande
+ */
+
+public class DDLFieldGeoView extends LinearLayout implements DDLFieldViewModel<GeolocationField>, TextWatcher {
+
+
+	public static final double INVALID_LATITUDE = -91;
```
Comment (nhpatt) [inline android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java]: ?

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+	@Override
+	public GeolocationField getField() {
+		return field;
+	}
+
+	@Override
+	public void setField(GeolocationField field) {
+		this.field = field;
+
+		if (this.field.isShowLabel()) {
```
Comment (nhpatt) [inline android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java]: if (labelTextView != null) {
  labelTextView.setVisibility(this.field.isShowLabel() ? VISIBLE : GONE);
labelTextView.setText(this.field.getLabel());
}
?

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+	@Override
+	public View getParentView() {
+		return parentView;
+	}
+
+	@Override
+	public void setParentView(View view) {
+		this.parentView = view;
+	}
+
```
Comment (nhpatt) [inline android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java]: <enter>

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java (hunk 1 above)
Comment (victorg1991) [reply android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java]: I inlined the value, but it was very "magic"

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java (hunk 3 above)
Comment (victorg1991) [reply android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java]: What happens here?

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java (hunk 1 above)
Comment (nhpatt) [reply android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java]: Why not -90 or -180/2 and +1/-1

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java (hunk 3 above)
Comment (nhpatt) [inline android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java]: formatting, 2 enters

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+	public void afterTextChanged(Editable s) {
+
+	}
+
+	private double castToDoubleOrDefault(String value, double defaultValue) {
+		if (value.isEmpty()) {
+			return defaultValue;
+		}
+
+		if ("-".equals(value)) {
```
Comment (nhpatt) [inline android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java]: ||

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java (hunk 2 above)
Comment (victorg1991) [reply android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldGeoView.java]: I'll buy it :P

**Comment by victorg1991** [comment]
Done! :D

**Pull request merged**